### PR TITLE
Tests: pam_lastlog unable to reset locked account

### PR DESCRIPTION
**Pull request by aborah-sudo**
https://bugzilla.redhat.com/show_bug.cgi?id=1285547

### CODE REVIEW

File: multihost_test/Bugzillas/test_Regression.py [hunk 1]
```diff
@@ -464,3 +471,63 @@ def test_bz450262(self, multihost, create_backup):
...
+        user = "test1"
+        opt = ["before", "help", "root", "time", "user", "set", "clear"]
+        client.run_command("useradd test1")
+        client.run_command(f"> {log}")
+        client.run_command(f"restorecon -vF {log}")
+        client.run_command(f"lastlog --help | grep '^[[:blank:]]*-[[:alpha:]],' > {options}")
+        real_data = client.run_command(f"cat {options}").stdout_text
+        for data in opt:
+            assert data in real_data
+        assert int(client.run_command(f"cat {options}  | wc -l").stdout_text.split()[0]) == len(opt)
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:512]: I think this should go in a separate test case.

File: multihost_test/Bugzillas/test_Regression.py [hunk 2]
```diff
@@ -464,3 +471,63 @@ def test_bz450262(self, multihost, create_backup):
...
+        :title:pam_lastlog unable to reset locked account
+        :id: 94626b58-f7c6-11ee-bf95-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1285547
+        :steps:
+          1. Creates a new user test1.
+          2. Clear the contents of the lastlog file and sets its context using restorecon.
+          3. Generates options for the lastlog command and checks if they are all present.
+          4. Simulates a login for user test1 that is five days in the past using log_back_in_time.
+          5. Checks the last login entry for test1 to ensure it's not showing "Never logged in".
+          6. Tests various options of lastlog command (--before, --time).
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:487]: You should create different explanations for each of the `before` and `time` runs. Take into account that 2 of them should succeed, but the other 2 should fail.

File: multihost_test/Bugzillas/test_Regression.py (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_Regression.py:522]: This contains automation for two bzs

File: multihost_test/Bugzillas/test_Regression.py [hunk 3]
```diff
@@ -464,3 +471,73 @@ def test_bz450262(self, multihost, create_backup):
...
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1285547
+                   https://bugzilla.redhat.com/show_bug.cgi?id=1114081
+        :steps:
+          1. Creates a new user test1.
+          2. Clear the contents of the lastlog file and sets its context using restorecon.
+          3. Generates options for the lastlog command and checks if they are all present.
+          4. Simulates a login for user test1 that is five days in the past using log_back_in_time.
+          5. Checks the last login entry for test1 to ensure it's not showing "Never logged in".
+          6. Lastlog command is run to check if the user (represented by the {user} variable)
+            has logged in before 4 days ago.
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:489]: Change by `Run lastlog to check if the user has logged in before the last 4 days.`

File: multihost_test/Bugzillas/test_Regression.py [hunk 4]
```diff
@@ -464,3 +471,73 @@ def test_bz450262(self, multihost, create_backup):
...
+          7. Check if an exception is raised when attempting to query login
+            information before 6 days ago for the specified user.
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:491]: Change by `Run lastlog to check if the user has logged in before the last 6 days.`

File: multihost_test/Bugzillas/test_Regression.py [hunk 5]
```diff
@@ -464,3 +471,73 @@ def test_bz450262(self, multihost, create_backup):
...
+          8. Check if an exception is raised when using the --time option instead
+            of the --before option with a value of 4 days.
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:493]: Change by `Run lastlog to check if the user has logged in the last 4 days.`

File: multihost_test/Bugzillas/test_Regression.py [hunk 6]
```diff
@@ -464,3 +471,73 @@ def test_bz450262(self, multihost, create_backup):
...
+          9. Lastlog command is executed to check if the user has logged in before 6 days ago.
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:494]: Change by `Run lastlog to check if the user has logged in the last 6 days.`

**Review by ikerexxe — APPROVED** [review]
LGTM!

**Pull request merged**